content/fa/_index.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
I think 
content/fa/_index.**md** should be content/fa/_index.**html**

considering content/en/_index.**html**